lib/WooCommerce/TransactionalEmailHooks.php

```diff
@@ -37,7 +37,7 @@ public function useTemplateForWoocommerceEmails() {
       $this->wp->removeAction('woocommerce_email_header', $emailHeaderCallback);
       $this->wp->removeAction('woocommerce_email_footer', $emailFooterCallback);
       $this->wp->addAction('woocommerce_email_header', function($emailHeading) {
-        $this->renderer->render($this->getNewsletter());
+        $this->renderer->render($this->getNewsletter(), $emailHeading);
```

Comment:
`$emailHeading` is not the email subject, but the [heading](https://github.com/woocommerce/woocommerce/blob/master/templates/emails/email-header.php#L48) used in an email body. Not exactly what the task requires, but still does the job. What do you think?

Reply:
I have discussed this topic with Pavel because I wanted to use the subject, but we didn't find a possible solution. In my opinion, the current solution is good. I would update a specification in the ticket.

Reply:
I've updated the ticket spec.

Reply:
> I have discussed this topic with Pavel
If you were aware of that issue beforehand, you could mention this in the PR to avoid the back-and-forth.

Reply:
Thank you. This is my fault. I'm sorry.